### DISCUSSION TITLE
Outstanding for next release

### DISCUSSION THREAD

**Discussion by lewri** [CorsixTH/CorsixTH#1856] [Reception]
It would be good I think to make a plan on stuff flagged for 0.65 and what ideally should be completed before the next release and what can wait.

The following issues are tagged for 0.65 currently outstanding:
Request | Description | Notes | Decision
---- | ----------- | -------- | ---------
#526 | Placing receptionists on desk | **[Has PR #1801]** | Fixed
#1628 | ISO choice in Installer | **[Has PR #1660]**
#1633 | Moving system info to gamelog | **[Has PR #1833]**
#1645 | Stuck doctors (missed 0.64) | **[Has PR #1763]** | Fixed
#1668 | Patients not always responding to completed operating theatre |**[Has PR #1875]** | Fixed
#1676 | Gatekeeper error on macOS Catalina+ | (Likely have a workaround for this again) | Use workaround?
#1682 | Queue:RereouteAllPatients() can glitch out | | Hold back (only found by manipulation of a PR)
#1715 | Prevent faxes opening on pause | **[Has PR #1757]** | Fixed
#1838 | False-positive missing Psychiatrist Fax | **[Has PR #1851]** | Fixed

The following pull requests are tagged for 0.65 currently outstanding:
Request | Description | Linked Issue(s) | Status | Decision
---- | ----------- | --------------- | ------- | ---------
#1660 | Drop Hospital Directory for installer | **[Fixing #1628]** | _Status RFC_
#1681 | Better unmanned receptionist handling | **[Fixing #1673]** | _Status RFC_ (Also blocked by 1682) | Hold back
#1697 | Meta Request fixes for updates, api, fixes | | _Status WIP_ (But has been broken into smaller PRs)
#1706 | Patients stay in ward if nurse leaves | **[Fixing #1277]** | _Status RDY_  | 
#1708 | Adjustment to gates_to_hell placement | Fixed | Merged for 0.65 | Merged
#1757 | Stop faxes being openable on pause | **[Fixing #1715]** | Fixed | Merged
#1763 | Maintain queue push-priority order | **[Fixing #1645]** | _Status RFC_ | Merged
#1794 | Serious Radiation graphics | **[Fixing #1659]** | _Status Unknown_ | Hold back
#1833 | Moving system info and gamelog location | **[Fixing #1633]** | _Status Unknown_

Thankfully I don't think we have anything currently outstanding that would be game-breaking. Though there are potentially bugs that would cause temporary disruption (such as the false-positive faxes). If any PRs or issues should be tagged to be complete ideally for 0.65 it would be helpful to add them: https://github.com/CorsixTH/CorsixTH/projects/1

**Comment by lewri** [comment]
I forgot toby's PR #1801 (Place receptionists on desk) that is currently RDY.

**Reply by lewri** [reply]
Now merged.

**Comment by TheCycoONE** [comment]
My #1660 - I'm open to further discussion on. It was the result of a conversation with a single individual but I'm not sure if removing it just creates more confusion for everyone else or if the macos/linux startup experience is better for everyone.

Open to other decisions on that but I didn't merge it in the 0.64 release because I wasn't confident it was the right thing to do.

**Comment by TheCycoONE** [comment]
It should be noted somewhere that we also need a lua-sec vcpkg port for the 0.65 release, thanks to the changes in the updater.

**Comment by TheCycoONE** [comment]
Thanks for this list @lewri - it's a good place to focus. I added some comments on #1277 - it's good except for save compatibility - had left it sitting too long.

**Comment by lewri** [comment]
@driverpt if you're about at the moment could you let us know if Serious Radiation could be in a ready state soon (probably about a month's time) and have enough time to check for any bugs?

**Reply by lewri** [reply]
Temporarily placed this into miss 0.65 unless we get some more info from driver

**Comment by mugmuggy** [comment]
With #1708 its ready. There is an issue which I was going to try and resolve around the IdleSpawnAction. The grim faces a particular direction, then that changes quickly to another direction before he starts walking. Thats the case before the change also so I'm not too concerned in fixing it.

**Reply by lewri** [reply]
Should it be added as a TODO if it will be fixed in the future?

**Reply by mugmuggy** [reply]
Probably ok to leave it as is, but you know once you seen it you can't unsee it.

**Comment by TheCycoONE** [comment]
We're getting close to the point where I try to publish a beta and ask for feature freeze.
